README.md

```diff
@@ -325,6 +325,36 @@ To reimport just one entity, run:
 bundle exec rails dfe:analytics:import_entity[entity_name]
 ```
 
+
+## Event debugging
+
+If you wish to log events for debug purposes, create a file `config/analytics_event_debug.yml` containing an array of your event filters  under a `shared` key like:
+
+```yaml
+shared:
+  event_filters:
+    -
+      type: (create|update|delete)_entity
```

Comment:
@asatwal what about import_entity events? Would they be supported too?

Reply:
So type can be absolutely anything. It's actually converted to a regular expressions (Which is the first bullet point: `All values are converted to regular expressions for matching`). So if we wanted to match on `import_entity`, we would just add another filter to the event_filters array in the YAM file:

``` YAML
  event_filters:
    -
      type: import_entity
```